src/compiler/crystal/macros.cr

```diff
@@ -1878,13 +1878,49 @@ module Crystal::Macros
   # class MacroLiteral < ASTNode
   # end
 
-  # if inside a macro
-  # class MacroIf < ASTNode
-  # end
+  # An `if` inside a macro, e.g.
+  #
+  # ```
+  # {% if cond %}
+  #   puts "Then"
+  # {% else %}
+  #   puts "Else"
+  # {% end %}
+  # ```
+  class MacroIf < ASTNode
+    # The condition of the `if` clause
+    def cond : ASTNode
+    end
 
-  # for inside a macro:
-  # class MacroFor < ASTNode
-  # end
+    # The code to be executed when the condition is truthy
```

Comment:
I don't think "execution" is the appropriate term here

Reply:
Evaluated, perhaps?

Reply:
You're right. "Printed" or "used" come to my mind as further alternatives. What's your suggestion @HertzDevil ?

Reply:
I think it's simply unnecessary to even explain the meaning of a `then` block here.
The documentation for `cond` does not say "The code that determines the truthiness". It simply describes it as "The condition of the `if` clause."
Let's to the same here:
```suggestion
    # The then branch of the `if` clause.
```